### ISSUE TITLE
[UWP][Accessibility] Name attribute is not defined for image under Agenda.JSON file.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3931]
# Platform
* UWP

# Version of SDK
master

# Details
Repro Steps:​
1. Launch the application.
2. New screen starts appearing. Navigate to Left pane JSOn files.​
3. Now select "Agenda.JSON" button. It will update the Middle and Right Pane. 
4. Set to scan mode by using Caps + Space keys.
5. Now, navigate to image section at right pane.
6. Verify alt of image.

Actual Result:
Name attribute is not defined for image appearing at right pane of Agenda.JSON file and due to this, screen reader user is unable to get the purpose of image control.

Expected Result:
There must be name attribute defined for Image control appearing at right pane of Agenda.JSON file so that screen reader user could get the context of image control.
